man/import.q.concourse.Rd

```diff
@@ -28,13 +31,18 @@
 	This function simply imports all full item wordings and assigns a \emph{meaningful item short handle} based on the file name.
```

Comment:
`This function imports all full item wordings and assigns \emph{handles} based on the file name.`

Once you've defined what handle is (in qmethod-package or elsewhere), there's no need to qualify it in all places (the user would understand that handles are, by the definition in this package, meaningful and short, otherwise one might think that meaningful item short handle is different from handle).